docusaurus/docs/reactnative/core-components/channel.mdx

```diff
@@ -436,6 +436,14 @@ The max allowable is 255, which when reached displays as `255+`.
 | ------ | ------- |
 | number | 255     |
 
+### giphyVersion
+
+The Giphy version to render - check the keys of the [Image Object](https://developers.giphy.com/docs/api/schema#image-object) for possible values.
+
+| Type   | Default        |
+| ------ | -------------- |
+| string | 'fixed_height' |
```

Comment:
Move this prop doc to separate file and you can reuse it within messages_context.mdx. Check `giphyEnabled ` prop for example